.github/actions/initialize/action.yml

```
@@ -9,6 +9,12 @@ description: >
 runs:
   using: "composite"
   steps:
+    - name: Init Info
+      shell: pwsh
+      run: |
+        Write-Output "Current Powershell environment:"
```

[The environment is something different.](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7.4)
```suggestion
        Write-Output "Current PowerShell host:"
```